src/logger/logger.ts

```diff
@@ -63,9 +63,17 @@ export default class Logger {
         (m, p1) => `${p1}.<sig>`,
       );
     };
+    
+    const truncateSecret = (data: string): string => {
+      return data.replace(
+        /,?"client_secret":"[a-zA-Z0-9_+=.@/-]{60,}"/gi, // so far I only saw secrets with 64 characters
+        '',
```

Comment:
Can we replace it with something like "REDACTED" instead, so that we don't think there was no secret "empty string" and we understand the logger has hidden it?

Reply:
wanna have it like "client_secret":"REDACTED"   or just "REDACTED" ?

Reply:
"client_secret":"REDACTED" seems like the least confusing to me